util/Cargo.toml

```diff
@@ -22,11 +22,11 @@ log = "0.4"
 walkdir = "2"
 zip = { version = "0.4", default-features = false }
 parking_lot = {version = "0.6"}
-zeroize = "0.5.2"
+zeroize = "0.9"
```

Comment:
Just noting Drop is implemented by `ZeroingString` in  `util/types.rs` so should be okay here.